vectorio/_rectangle.py

```diff
@@ -85,11 +86,11 @@ def height(self, value: int) -> None:
     @property
     def color_index(self) -> int:
         """The color_index of the rectangle as 0 based index of the palette."""
-        return self._color_index - 1
+        return self._color_index
```

Comment:
I think we need to keep the `- 1` here and the `+ 1` from a few lines below this. They match the core implementation here: https://github.com/adafruit/circuitpython/blob/main/shared-module/vectorio/Rectangle.c#L79-L84